src/frontend/src/app/myprofile/myprofile.component.ts

```diff
@@ -25,9 +24,9 @@ export class MyProfileComponent {
       medium: 10,
       hard: 10,
     } as PlayerData;
-    this.playerStats = authService.getPlayerData();
-    this.player.picture = this.playerStats.picture;
-    this.player.name = this.playerStats.name;
+    var playerStats = authService.getPlayerData();
```

Comment:
Use const instead of var.

The destructing syntax might also be useful for this i.e. const {picture, name} = authService.getPlayerData().

Reply:
Changed playerStats to const